### PR TITLE
Sclark/rubocop fixes add badge

### PR DESCRIPTION
**Pull request by sclark007**
* fix gem ordering for rubocop errors
* add travis badge to readme.

### CODE REVIEW

**Comment by jjasghar** [comment]
Seems reasonable, can you squash these commits down? 

With our new policy please use the [DCO signing process](https://discourse.chef.io/t/a-developer-certificate-of-origin-dco-is-now-required-with-code-contributions/9579).

For a [detailed description](http://stackoverflow.com/questions/13457203/how-to-add-the-signed-off-by-field-in-the-git-patch) or you can just do:

```
$ git commit -s
$ #Or
$ git commit --signoff
```

If you already have the commit, use git commit --amend or git rebase -i to edit your commit message, and add the above signoff line.

File: .travis.yml [hunk 1]
```diff
@@ -2,9 +2,9 @@ language: ruby
 cache: bundler
 sudo: false
 rvm:
-- 2.0.0
-- 2.1
-- 2.2
+  - 2.1.10
```
Comment (jjasghar) [inline .travis.yml:5]: Please remove this from the `.travis.yml`

**Comment by sclark007** [comment]
@jjasghar pretty sure i made things worse. this was my first attempt at using git to squash a commit. i can close the pull request and start over if needed. 

thanks,

**Comment by jjasghar** [comment]
Closing due to the confusion. No worries though thanks for the PR, if you want to try again I'm more than willing to review another time.

**Pull request closed without merging**